test/07-openqa-label-known-issues.t

```diff
@@ -42,8 +43,18 @@ out=''
 hxnormalize() {
     cat "$2"
 }
+
 hxselect() {
-    cat -
+    if [ -p /dev/stdin ]; then
+	myinput="$(cat)"
+	if echo "$myinput" | grep -q "incomplete"; then
+	    echo "$myinput" | awk -F'title="' '{split($2, a, "\""); print a[1]}'
+	else
+	    cat -
+	fi
+    else
+	cat -
+    fi
```

Comment:
We are using spaces in this file for indentation. You could also avoid the duplicate branches:

```suggestion
    if [[ -p /dev/stdin ]]; then
        myinput=$(cat)
        if echo "$myinput" | grep -q "incomplete"; then
            echo "$myinput" | awk -F'title="' '{split($2, a, "\""); print a[1]}'
            return 0
        fi
    cat -
```